File: homework-management-csharp/LAB9-2/repository/NotaRepository.cs
using LAB9_2.domain;
using LAB9_2.validation;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace LAB9_2.repository
{
    class NotaRepository : AbstractCRUDRepository<KeyValuePair<Student, Tema>, Nota>
    {
        public NotaRepository(IValidator<Nota> validator) : base(validator) { }
    }
}

File: homework-management-java/src/com/company/repository/NotaRepository.java
package com.company.repository;

import com.company.domain.*;
import com.company.validation.*;

import java.io.BufferedWriter;
import java.io.FileWriter;
import java.io.IOException;

public class NotaRepository extends AbstractCRUDRepository<Pair<String, String>, Nota> {
    public NotaRepository(Validator<Nota> validator) {
        super(validator);
    }
}
